rules/evasion/fake-process-name.yara

```diff
@@ -7,10 +6,14 @@ rule fake_kworker_val : critical {
     hash_2023_Unix_Downloader_Rocke_6107 = "61075056b46d001e2e08f7e5de3fb9bfa2aabf8fb948c41c62666fd4fab1040f"
   strings:
     $kworker = /\[{0,1}kworker\/[\d:\]]{1,5}/
-    $kworker2 = "kworker" fullword
+    $kworker2 = /\bkworker\b/
```

Comment:
just for my own education, what's the difference between \b and fullword?

Reply:
For the purposes I was using it for, probably not. It would definitely provide more control in a larger expression but I can change this back to `"kworker" fullword`.